docs/fwupd.conf.md

```diff
@@ -0,0 +1,268 @@
+---
+title: Configuration File Format
+---
+
+% fwupd.conf(5) @PACKAGE_VERSION@ | Configuration File Format
+
+NAME
+----
+
+**fwupd.conf** — configuration file for the fwupd daemon.
+
+SYNOPSIS
+--------
+
+The `/etc/fwupd/fwupd.conf` file is the main configuration file for the fwupd daemon.
```

Comment:
as we'll be processing this, I think you can technically populate prefix here too.

Reply:
Good idea, fixed.